whipper/test/test_common_mbngs.py

```diff
@@ -1,5 +1,5 @@
 # -*- Mode: Python; test-case-name: whipper.test.test_common_mbngs -*-
-# vi:si:et:sw=4:sts=4:ts=4
+# vi:si:et:sw=4:sts=4:ts=4:set fileencoding=utf-8
```

Comment:
The encoding is required to be defined due to the “Ö” in the comment to the new test.